module/PowerShellEditorServices/Start-EditorServices.ps1

```diff
@@ -46,7 +46,7 @@ param(
     [ValidateNotNullOrEmpty()]
     $LogPath,
 
-    [ValidateSet("Diagnostic", "Normal", "Verbose", "Error")]
+    [ValidateSet("Diagnostic", "Normal", "Verbose", "Warning", "Error")]
```

Comment:
One minor suggestion - could you change the order of `Normal` and `Verbose` since the log level order is actually: Diagnostic, Verbose, Normal, Warning, Error.  I don't think it really matters that much but might as well have the correct order here since it is already close to the correct order.  :-)

Reply:
sure, done.

Reply:
Thanks!